### PR TITLE
Total running jobs quota

### PR DESCRIPTION
**Pull request by romasku**

### CODE REVIEW

File: neuromation/api/admin.py [hunk 1]
```diff
@@ -131,8 +132,9 @@ async def set_user_quota(
         self,
         cluster_name: str,
         user_name: str,
-        gpu_value_minutes: Optional[float],
-        non_gpu_value_minutes: Optional[float],
```
Comment (romasku) [inline neuromation/api/admin.py:135]: Quotas are ints everywhere (on server, in cli parser), so I think `Optional[float]` was a mistake.

File: neuromation/cli/job.py [hunk 2]
```diff
@@ -270,6 +270,12 @@ def job() -> None:
     show_default=True,
     help="Upload neuro config to the job",
 )
+@option(
+    "--wait-for-job-quota/--no-wait-for-job-quota",
```
Comment (asvetlov) [inline neuromation/cli/job.py:274]: The name is quite long.
I have no idea what should be better.
Maybe @mariyadavydova can suggest something?

Comment (romasku) [reply neuromation/cli/job.py:274]: I can suggest the following alternatives:

`--wait-for-quota`
`--queue-job`
`--fail-on-quota/--no-fail-on-quota`
`--keep-pending`
`--no-quota-error`
`--wait-for-seat`
`--run-eventually`

But as for me, none of them are good.

Comment (mariyadavydova) [reply neuromation/cli/job.py:274]: I like `--wait-for-seat`.

Comment (asvetlov) [reply neuromation/cli/job.py:274]: Ok, let's rename to `--wait-for-seat` and merge

**Review by asvetlov — APPROVED** [review]

**Pull request merged**